easybuild/easyconfigs/f/FlexiBLAS/FlexiBLAS-3.0.4-GCC-10.2.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'FlexiBLAS'
+version = '3.0.4'
+
+homepage = 'https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release'
+description = """FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation
+used by a program without recompiling or relinking it."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://csc.mpi-magdeburg.mpg.de/mpcsc/software/flexiblas/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['50a88f2e88994dda91b2a2621850afd9654b3b84820e737e335687a46751be5c']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
```

Comment:
Just a note that FlexiBLAS autodetection with CMake did not appear until CMake 3.19

Reply:
I'll bump this to `GCC/10.3.0` + `OpenBLAS/0.3.15` + `CMake/3.20.1` for `2021a`, it doesn't make much sense to have this for `2020b` anyway...